README.md

```diff
@@ -128,8 +130,30 @@ Alternatively, you can use the [CLI](#kogito-cli) to deploy your services:
 $ kogito deploy-service example-quarkus https://github.com/kiegroup/kogito-examples/ --context-dir=drools-quarkus-example
 ```
 
+### Clean up a Kogito Service deployment
+
+```bash
+$ kogito delete-service example-quarkus
+```
+
+### Native X JVM Builds
+
+By default, the Kogito Services will be built with traditional `java` compilers to speed up the time and save resources. This means that the final generated artifact will be a [uber jar](https://stackoverflow.com/questions/11947037/what-is-an-uber-jar) with the chosen runtime (default to Quarkus).
+
+Kogito Services when implemented with [Quarkus](https://quarkus.io/guides/kogito-guide) can be built to native binary. This means low ([really low](https://www.graalvm.org/docs/examples/java-performance-examples/)) footprint on runtime, but will demand a lot of resources during build time. Read more about AOT compilation [here](https://www.graalvm.org/docs/reference-manual/aot-compilation/).
+
+In our tests, native builds takes approximately 10 minutes and the build pod can consume up to 10GB of RAM and 1.5 CPU cores. Make sure that you have this resources available when running native builds.
```

Comment:
is these values retrieved from a real example? would be better add some print or logs(top)?

Reply:
Yes, I spent the whole day yesterday building our samples, Maciej's travel agency and our rain-forecast service. I don't have prints in hand, but personally I don't think that this would be add much. Look at this one: https://pasteboard.co/IDk0Ps8.png